config-generator/src/generate.js

```diff
@@ -169,6 +169,10 @@ function prometheusConfig(params) {
         region: params.region
       },
     },
+    queue_config: {
+      max_samples_per_send: 4000,
+      batch_send_deadline: '30s'
+    },
```

Comment:
My [previous changes](https://github.com/livepeer/livepeer-monitoring/pull/164/files) didn't change the -[requests rate](https://eu-metrics-monitoring.livepeer.monster/grafana/goto/EGwUuQFSR?orgId=1) too much.
I think we can delay sending metrics from default `5s` to `30s` and `max_samples_per_send` from `500` to `4000`.

https://prometheus.io/docs/practices/remote_write/
https://prometheus.io/docs/prometheus/latest/configuration/configuration/#remote_write